test/xpu_api/numerics/numeric.ops/numeric.ops.gcd/xpu_gcd.pass.cpp

```diff
@@ -0,0 +1,150 @@
+// -*- C++ -*-
+//===-- xpu_gcd.pass.cpp --------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include <iostream>
+
+#include <CL/sycl.hpp>
+#include <oneapi/dpl/numeric>
+#include <oneapi/dpl/type_traits>
+
+constexpr cl::sycl::access::mode sycl_read = cl::sycl::access::mode::read;
```

Comment:
`cl::` is not needed now. It can be just `sycl::...`

Reply:
Thanks for your comments. Done.